[Update] Installation and Update content to include MacPorts method (#816)
timescale/docs: install/installation-macos.md

# Install self-hosted TimescaleDB on macOS systems
You can host TimescaleDB yourself on your Apple macOS system.

> These instructions use a Homebrew or Macports installer on these versions:

Reviewer: ```suggestion
These instructions use a Homebrew or MacPorts installer on these versions:
```

</procedure>

## Install self-hosted TimescaleDB using Macports
You can use Macports to install TimescaleDB on macOS-based systems.

<procedure>

### Installing self-hosted TimescaleDB using Macports
1.  Install MacPorts, if you don't already have it. For more information about
    MacPorts, including installation instructions, see the
    [MacPorts documentation][macports].
1.  Install TimescaleDB:
    ```bash
    sudo port install timescaledb
    ```
1.  **OPTIONAL**: View the files that have been installed:
    ```bash
    port contents timescaledb
    ```

<highlight type="important">
By default, MacPorts does not install the `timescaledb-tools` package, so the
`timescaledb-tune` script is not included. For more information about installing
and using the tool, see the

> [timescaledb-tune](/timescaledb/:currentVersion:/how-to-guides/configuration/timescaledb-tune/#timescaledb-tuning-tool) section.

Reviewer 2: Oh, good spot! I got halfway there 😉

</procedure>

<procedure>

#### Using MacPorts

> 1.  Install MacPorts, by downloading the macOS package and double-click the downloaded package installer.

Reviewer 3: ```suggestion
1.  Install MacPorts by downloading and running the package installer.
```

    
    For more information about MacPorts, including installation instructions,
    see the [MacPorts documentation][macports].
1.  Install TimescaleDB:
    ```bash
    sudo port install timescaledb 
    ```

> 1.  View the files that were installed by timescaledb:

Reviewer 3: ```suggestion
1.  View the files that were installed by TimescaleDB:
```

    ```bash
    port contents timescaledb
    ``` 
     
<highlight type="warning">

>  MacPorts does not install the `timescaledb-tools` to run the `timescaledb-tune` script. For more information about installing and using the tool, see [timescaledb-tune][tune] section.

Reviewer 3: ```suggestion
 MacPorts does not install the `timescaledb-tools` to run the `timescaledb-tune` script. 
 For more information about installing and using the tool, see the [timescaledb-tune][tune] section.
```
